_layouts/category-archive.html

```diff
@@ -0,0 +1,21 @@
+---
+layout: default
+---
+
+<div class="post">
+
+  <header class="post-header">
+    <h1 class="post-title"> <i class="fas fa-tag fa-sm"></i> {{ page.title }} </h1>
+    <p class="post-description"> an archive of posts in this category </p>
+  </header>
+
+  <article>
+     {% for post in page.posts %}
+    <li>
+      <span class="post-date">{{ post.date | date: "%b %-d, %Y" }}</span>
+      <a class="post-link" href="{{ post.url | relative_url }}">{{ post.title }}</a>
+    </li>
```

Comment:
do you mind changing this from a simple list to a table-like structure? basically, identical to how `news` are displayed on the about page.